CvGameCoreDLL_Expansion2/CvImprovementClasses.cpp

```diff
@@ -801,6 +833,47 @@ int CvImprovementEntry::GetAdditionalUnits() const
 }
 #endif
 
+/// Bonus yield if another improvement is adjacent
+fraction CvImprovementEntry::GetYieldPerXAdjacentImprovement(YieldTypes eYield, ImprovementTypes eImprovement) const
+{
+	CvAssertMsg(eImprovement < GC.getNumImprovementInfos(), "Index out of bounds");
+	CvAssertMsg(eImprovement > -1, "Index out of Bounds");
+	CvAssertMsg(eYield < NUM_YIELD_TYPES, "Index out of bounds");
+	CvAssertMsg(eYield > -1, "Index out of bounds");
+
+	fraction fYield = 0;
+	map<YieldTypes, map<ImprovementTypes, fraction>>::const_iterator itImprovement = m_YieldPerXAdjacentImprovement.find(eYield);
+	if (itImprovement != m_YieldPerXAdjacentImprovement.end())
+	{
+		map<ImprovementTypes, fraction>::const_iterator itYield = itImprovement->second.find(eImprovement);
+		if (itYield != itImprovement->second.end())
+		{
+			fYield = itYield->second;
+		}
+	}
+
+	// Special case for culture
+	if (eYield == YIELD_CULTURE)
+	{
+		fYield += m_iCultureAdjacentSameType;
+	}
+
+	return fYield;
+}
+bool CvImprovementEntry::IsYieldPerXAdjacentImprovement(YieldTypes eYield) const
+{
+	CvAssertMsg(eYield < NUM_YIELD_TYPES, "Index out of bounds");
+	CvAssertMsg(eYield >= -1, "Index out of bounds");
+	
+	if (eYield == NO_YIELD)
+		return (!m_YieldPerXAdjacentImprovement.empty() || m_iCultureAdjacentSameType != 0);
+	else if (eYield == YIELD_CULTURE)
+		return m_iCultureAdjacentSameType != 0;
```

Comment:
this needs to be:
```
if (eYield == YIELD_CULTURE && m_iCultureAdjacentSameType != 0)
    return true;
```
as it is now, normal culture adjacency yields don't work at all

Reply:
Removed m_iCultureAdjacentSameType entirely